### PR TITLE
Include AF option in diagnosing problems page

### PR DESCRIPTION
**Pull request by chwilliams9487**
**- Summary**
Adding an option for the Automation Framework using the log community script (https://github.com/zaproxy/community-scripts/blob/main/httpsender/LogMessages.js) on the diagnosing problems page

**- Test plan**
N/A, documentation only

**- Description for the changelog**

Added option for Automation Framework in Diagnosing Docker Problems  

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: site/content/docs/docker/diagnosing-problems.md [hunk 1]
```diff
@@ -118,3 +118,30 @@ If you are using the ZAP daemon in docker and controlling it via the API then yo
...
 
 All of the requests and responses made by or proxied through ZAP will be written to the `req-resp-log.txt` file in your CWD as above.
+
+### Automation Framework
+
+If you are using the Automation Framework then you will need to:
+
+1. Copy the following file to your CWD
+   * [LogMessages.js](https://github.com/zaproxy/community-scripts/blob/main/httpsender/LogMessages.js)
+2. Modify your YAML plan file to include this script as a parameter, e.g.
```
Comment (thc202) [inline site/content/docs/docker/diagnosing-problems.md:128]:
```suggestion
2. Modify your YAML plan file to add the script, e.g.
```

**Comment by thc202** [comment]
Commits can be squashed.

**Comment by kingthorin** [comment]
@chwilliams9487 I'm guessing you're using the Web UI for this PR so let me know if you need help with flattening this.

**Comment by chwilliams9487** [comment]
> @chwilliams9487 I'm guessing you're using the Web UI for this PR so let me know if you need help with flattening this.

@thc202 @kingthorin Commits are squashed, let me know if I can provide anything further for this!

**Comment by thc202** [comment]
You can remove the whole second part in the commit message, not needed for the history.

**Comment by chwilliams9487** [comment]
Resolved

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**